### ISSUE TITLE
The Storm Form spell has no description (also the school is empty)

### ISSUE THREAD

**Issue by abramche** [crawl/crawl#4080]
**PLATFORM and OS or BROWSER:**
CDO trunk console

**DCSS VERSION AND GAME INFORMATION:**
0.33-a0-218-g443864507a

**DESCRIPTION:**
1. Go to the spell (M)emorisation page, find the spell and check its description

```
Storm Form

This spell has no description. Casting it may therefore be unwise. Please file a
bug report.
Level: 7        School:         Fail: 100%

Power: N/A
Range: N/A
Noise: A bit loud
Miscasting this spell causes magic contamination and also .
```

**EXTRA FILES:**
http://crawl.develz.org/saves/dumps/hayenne-443864507a-241007-1608.tar.bz2

**Comment by matthewgranstrom** [comment]
This is a removed spell -- All the 'form' spells were replaced by talismans. How did your character find a way to access storm form? If you upgraded from a version older than talismans, that's the likely culprit, otherwise if you found it in a book, which vault placed that book?

**Comment by abramche** [comment]
Yes, I started on an older version and upgraded the version online. I suppose the easy solution would be to just remove deprecated spells from the library on version upgrade?

**Comment by DracoOmega** [comment]
I am not precisely sure at what point this changed, but currently upgrading saves with deprecated spells in the library does appear to properly hide/remove them.
